README.md

```diff
@@ -108,41 +247,14 @@ test('Some test case', async function (assert) {
 
   await visit('/');
   await a11yAudit('.modal', axeOptions);
-  assert.ok(true, 'no a11y errors found!');
-});
-```
-
-#### Integration and Unit Tests
-
-The helper is also able to be used Integration/Unit tests like so:
-
-```javascript
-import a11yAudit from 'ember-a11y-testing/test-support/audit';
-
-// ...elided for brevity
-
-test('Some test case', function (assert) {
-  this.render(hbs`{{some-component}}`);
 
-  let axeOptions = {
-    rules: {
-      'button-name': {
-        enabled: false,
-      },
-    },
-  };
-  return a11yAudit(this.element, axeOptions).then(() => {
-    assert.ok(true, 'no a11y errors found!');
-  });
+  assert.ok(true, 'no a11y errors found!');
```

Comment:
Hehe, this really points out that the `a11yAudit` function throws 😭.

I very much want to change it to return something instead...

Reply:
Ya. I was thinking it would be nice to have an assertion API for when you don't have any other assertions in your test.

`assert.a11y().noViolationsExist();` and `assert.a11y().violationsExist();` or some such thing